### PR TITLE
fix(server): change the arthas bind ip 

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
image

sometime arthas test failed, see https://github.com/apache/incubator-hugegraph/actions/runs/7652616070/job/20852745753?pr=2408

### CODE REVIEW

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java [hunk 1]
```diff
@@ -294,7 +294,7 @@ public static synchronized ServerOptions instance() {
                     "arthas.ip",
                     "The IP provided by Arthas, it can be accessible from the outside.",
                     disallowEmpty(),
-                    "0.0.0.0"
+                    "127.0.0.1"
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java:297]: any context for it? change the IP to localhost

Comment (SunnyBoy-WYH) [reply hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java:297]: the context you can see the introduction about this issue. i think 0.0.0.0 means any ip, and arthas will set default password, sometime the ci failed.  

why prefer localhost not 127.0.0.1? the config item provided by arthas named 'arthas.ip'? im not sure localhost as a domain will work.

File: hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/api/ArthasApiTest.java [hunk 2]
```diff
@@ -43,16 +43,9 @@ public void testArthasApi() {
...
-        // and if we are not set auth when arthas attach hg, arthas will auth it and return 401.
-        // ref:https://arthas.aliyun.com/en/doc/auth.html#configure-username-and-password
         Response r = arthasApiClient.post(ARTHAS_API_PATH, body);
         String result = assertResponseStatus(200, r);
         assertJsonContains(result, "state");
         assertJsonContains(result, "body");
-
-        RestClient arthasApiClientWithAuth = new RestClient(ARTHAS_API_BASE_URL);
-        r = arthasApiClientWithAuth.post(ARTHAS_API_PATH, body);
-        assertResponseStatus(401, r);
```
Comment (javeme) [inline hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/api/ArthasApiTest.java:56]: can we add some exception cases for invalid requests

Comment (SunnyBoy-WYH) [reply hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/api/ArthasApiTest.java:56]: Yes we will do

**Pull request closed without merging**